ndcube/extra_coords/table_coord.py

```diff
@@ -56,18 +64,21 @@ def _generate_tabular(lookup_table, interpolation='linear', points_unit=u.pix, *
     if len(points) == 1:
         points = points[0]
 
-    kwargs = {
-        'bounds_error': False,
-        'fill_value': np.nan,
-        'method': interpolation,
-        **kwargs
-    }
-
-    t = TabularND(points, lookup_table, **kwargs)
-
-    # TODO: Remove this when there is a new gWCS release
-    # Work around https://github.com/spacetelescope/gwcs/pull/331
-    t.bounding_box = None
+    if len(lookup_table) == 1:
+        t = length1_lookup_table(lookup_table, fill_value=kwargs.get("fill_value", np.nan))
+    else:
+        kwargs = {
+            'bounds_error': False,
+            'fill_value': np.nan,
+            'method': interpolation,
+            **kwargs
+        }
+
+        t = TabularND(points, lookup_table, **kwargs)
+
+        # TODO: Remove this when there is a new gWCS release
```

Comment:
Is this done now?

Reply:
Looks like it was included in the 0.16.0 gWCS release.  If we remove this now, we'll have to pin ndcube 2.0.1 to that gWCS version, right?

Reply:
Given 0.16 is nearly a year old, I think we could do that?